cdapython/utils/utility.py

```diff
@@ -101,23 +101,49 @@ def get_table_version() -> str:
 def unique_terms(
     col_name: str,
     system: str = "",
-) -> Q:
+    offset: int = 0,
+    host: Optional[str] = None,
+    verify: Optional[bool] = None,
+    async_req: Optional[bool] = True,
+    show_sql: bool = False,
+    show_counts: bool = False,
+    verbose: bool = True,
+    limit: int = 100,
+) -> Paged_Result:
+    # TODO : Should the value be changed? The "show_counts" parameter was copied from Swagger-generated code. Is this description sufficient?
     """
     Show all unique terms for a given column.
     Args:
-        col_name (str): _description_
-        system (str, optional): _description_. Defaults to "".
+        col_name (str): This is the default way to search for a unique term from the CDA service API.
```

Comment:
Thanks for filling out the docstring!  It's been very helpful to guide our conversations.